homeassistant/components/steam_online/sensor.py

```diff
@@ -140,10 +147,28 @@ def _get_current_game(self):
         _LOGGER.error("Unable to find name of app with ID=%s", game_id)
         return repr(game_id)
 
+    def _get_last_online(self):
+        last_online = datetime.fromtimestamp(
+            mktime(self._profile.last_online)
+        ).isoformat()
+
+        if last_online:
+            return last_online
+
```

Comment:
Please return `None` explicitly here. If a function returns something somewhere it should return explicitly in all cases.